Dockerfile

```diff
@@ -3,12 +3,12 @@
 FROM httpd:alpine
 
 RUN rm -r /usr/local/apache2/htdocs/*
-COPY /build /usr/local/apache2/htdocs/
+COPY /build /usr/local/apache2/htdocs/8.1/
```

Comment:
This Dockerfile is for local development. Running the site within an 8.1 folder allows us to not have to make changes to the docusaurus configuration, which is expecting the site to be deployed within an 8.1 folder.